deep_tempering/callbacks.py

```diff
@@ -485,8 +482,10 @@ def exchange(self, *args, **kwargs):
 
 class MetropolisExchangeCallback(BaseExchangeCallback):
   """Exchanges of hyperparameters based on Metropolis acceptance criteria."""
-  def __init__(self, exchange_data, swap_step, burn_in=None):
+  def __init__(self, exchange_data, swap_step=1, burn_in=1, coeff=1., hp_to_swap=None):
```

Comment:
`hp_to_swap` should not be a part of the default exchange procedure. The default exchange is as is. If user wants to create other exchange procedure it needs to write his own callback.